<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSafeChatModeWidget</class>
 <widget class="QWidget" name="VSafeChatModeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VSafeChatModeWidget</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请注意</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_txt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;识别到你的账号存在风险，现已限制聊天功能，请点击以下任意方式进行解锁&lt;br&gt;若无法完成认证解锁，请&lt;a href=&quot;kefu&quot;&gt;&lt;span style=&quot;; color:#0000ff;&quot;&gt;联系客服&lt;/span&gt;&lt;/a&gt;进行人工处理&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_checkMsg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>75</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>短信验证</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>75</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>身份认证</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_checkId">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_checkOrder">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>75</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>历史订单认证</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
